### PR TITLE
Added: FIT File Import: Option to set TourType to Sport Name or Profile Name from file v2

### PR DESCRIPTION
**Pull request by bbbbbr**
This PR integrates the PR feedback in #177 (including git author field). Let me know if I've missed anything or additional changes should be made.

Thanks!

Continuation of #176 and #177 : This allows the user to set the Tour Type based on the "Sport Name" or "Profile Name" when importing from Garmin FIT Files.

### CODE REVIEW

File: bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/IPreferences.java [hunk 1]
```diff
@@ -29,11 +29,11 @@
...
+   String FIT_IS_IMPORT_TOURTYPE_MODE           = "FIT_IS_IMPORT_TOURTYPE_MODE";          //$NON-NLS-1$
 
-   String FIT_IMPORT_TOURTYPE_MODE_SPORT           = "FIT_IMPORT_TOURTYPE_MODE_SPORT";              //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_PROFILE         = "FIT_IMPORT_TOURTYPE_MODE_PROFILE";            //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_TRYPROFILE      = "FIT_IMPORT_TOURTYPE_MODE_TRYPROFILE";         //$NON-NLS-1$
-   String FIT_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE = "FIT_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE";    //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_SPORT           = "FIT_IS_IMPORT_TOURTYPE_MODE_SPORT";              //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_PROFILE         = "FIT_IS_IMPORT_TOURTYPE_MODE_PROFILE";            //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_TRYPROFILE      = "FIT_IS_IMPORT_TOURTYPE_MODE_TRYPROFILE";         //$NON-NLS-1$
+   String FIT_IS_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE = "FIT_IS_IMPORT_TOURTYPE_MODE_SPORTANDPROFILE";    //$NON-NLS-1$
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/IPreferences.java]: `FIT_IMPORT_TOURTYPE` --> `FIT_IS_IMPORT_TOURTYPE ` this is OK

`FIT_IMPORT_TOURTYPE_MODE_...`  --> `FIT_IS_IMPORT_TOURTYPE_MODE_...` this was not my suggestion as this is not a boolean value

Comment (bbbbbr) [reply bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/IPreferences.java]: My misunderstanding, it's fixed

**Comment by telemaxx** [comment]
i am not sure how the new fit import function  works, when using "easy import"

today i merged the latest master into my master, so i could test the new fit import feature.
btw, great idea
the first think i noticed was, that my easy import configurations where gone.
maybe this was my fault, i am not sure. because i also re-imported all MT projects in eclipse.

? what does "ONE" mean, is this new? never recognized that.

?? when set a tourtype here and set also the new automatic in the preferences, who wins?

grafik

**Comment by wolfgang-ch** [comment]
> ? what does "ONE" mean, is this new? never recognized that.

Even I didn't know it but reading the tooltip may help :-)


mt-easy-import


>?? when set a tourtype here and set also the new automatic in the preferences, who wins?

the better :-)

easy import is an post processing feature, so it will win (I just testet it)

**Pull request merged**